### DISCUSSION TITLE
how is it different from authentik and authelia like solutions

### DISCUSSION THREAD

**Discussion by Greatz08** [zitadel/zitadel#7675] [General]
Please explain in detail if possible

**Comment by fforootd** [comment]
Uhm it depends on what you are looking for 😁 and I am clearly biased

But honestly I do not know both projects well enough to give you a detailed answer.
From a quick glance the main differences though are:

- Zitadel is better in case where you have multi-tenancy needs
- Zitadel has better audit features since it relies on event sourcing as storage layer
- Zitadel provides a session-api to build your own login-ui even with full openid-connect support
- Zitadel provides an event-api to filter and query all changes that happen
- Zitadel provides actions to run your own code to manipulate data or call external apis
- Authelia looks like it has a great support for proxy integrations (i.e. forward auth)
- Authelia and Zitadel both use Go, but differ in the UI, react and angular
- Authentik is written in python as it looks
- Authentik provides webhooks instead of actions
- Authelia does not look like it has an webhook/actions feature

With that sad I think **all of these projects** do a really great work on bringing people great turnkey solutions for auth, user management, sso....

The one thing I think we do differently with Zitadel is that we want to bring together devs, ops and security people in the same tool over time and become a platform tool rather then a point solutions that only solve a small piece of the problem.

Hope this helps
